themes/gatsby-theme-blog/src/theme/styles.js

```diff
@@ -0,0 +1,73 @@
+import prism from "./prism"
+
+export default {
+  root: {
+    fontFamily: `body`,
+  },
+  Container: {
+    maxWidth: 672,
+    px: 3,
+  },
+  pre: {
+    fontFamily: `monospace`,
+    tabSize: 4,
+    hyphens: `none`,
+    marginBottom: 0,
+    color: `white`,
+    bg: `none`,
+    overflow: `auto`,
+    p: 3,
+    ...prism,
+  },
+  code: {
+    fontFamily: `monospace`,
+    // from typography overrideThemeStyles
+    // "h1 code, h2 code, h3 code, h4 code, h5 code, h6 code"
+    fontSize: `inherit`,
+  },
+  // todo: prism seems to break MDX inlineCode
+  inlineCode: {
```

Comment:
I haven't yet looked into why this isn't working, but can either do that here or follow up with another PR

Reply:
Noted this on the main PR as a todo

Reply:
I *think* adding the `gatsby-remark-prismjs` `noInlineHighlight` option fixes this, but can confirm after merging